yarn.lock

```diff
@@ -1859,8 +1859,8 @@
     react-transition-group "^4.4.5"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.14.11"
-  resolved "https://github.com/mui/material-ui.git#7f9c09fea69759f0f621cbe3eb0f738d51c8201e"
+  version "5.14.16"
```

Comment:
I thought the previous design was gone until I discovered the view switch 😅

<img width="842" src="https://github.com/mui/mui-x/assets/13808724/bda3ea74-996e-4f9f-8d6f-ea9ae22e96ae">

Can we have the expanded list view enabled by default?
IMO it's a better default, because:
1. No need for horizontal scroll
2. "Link to the prop" button

Reply:
I'd be down to have the expanded list view as a default. 🤙 But we can make it in another focused/specific PR.

Reply:
> Can we have the expanded list view enabled by default?

We could but not for now. The layout is randomly chosen when you lang on the new design. And we track if users are modifying their layout. The idea is to be able to know many people moved from one layout to another one

Reply:
Is there a way to not make it random, though? Would that impact GA tracking data?

Reply:
If we remove the random aspect and set the same default one for everybody, it would block us from analyzing which view people prefer.
For me this is a temporary A/B test to determine which default value we will eventually set to everybody.

Reply:
That makes sense! 🤙